doc/dev_plugins.md

```diff
@@ -24,23 +25,35 @@ a_reader = "module_name:load_sketches"
 
 [project.entry-points."sourmash.save_to"]
 a_writer = "module_name:SaveSignatures_WriteFile"
+
+[project.entry-points."sourmash.cli_script"]
+new_cli = "module_name:Command_NewCommand"
 ```
 
 Here, `module_name` should be the name of the module to import.
-`load_sketches` should be a function that takes a location along with
+
+* `load_sketches` should be a function that takes a location along with
 arbitrary keyword arguments and returns an `Index` object
 (e.g. `LinearIndex` for a collection of in-memory
-signatures). `SaveSignatures_WriteFile` should be a class that
+signatures).
+* `SaveSignatures_WriteFile` should be a class that
 subclasses `BaseSave_SignaturesToLocation` and implements its own
 mechanisms of saving signatures. See the `sourmash.save_load` module
 for saving and loading code already used in sourmash.
+* `Command_NewCommand` should be a class that subclasses
+  `plugins.CommandLinePlugin` and provides an `__init__` and
+  `main` method.
 
-Note that if the function or class has a `priority` attribute, this will
-be used to determine the order in which the plugins are called.
+Note that if the reader function or writer class has a `priority`
+attribute, this will be used to determine the order in which the
+plugins are called.
```

Comment:
what guidelines do we want to have for priority? e.g. internal sourmash commands use priority xx - yy, external should use priority values above this, lower numbers are higher priority...

Reply:
adjusted in 6b034ae1. Still not perfect, but I think it needs more thought (and more examples) before we settle on numbers ;).